src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+img {
+  padding-left: 50px;
+  align-self: center;
+}
+
+header {
+  display: flex;
+  text-transform: uppercase;
+}
+
+nav {
+  display: flex;
+  margin-left: auto; /* Push nav to the right */
+  align-items: center;
+  margin-right: 50px;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  display: flex;
+  text-decoration: none;
+  color: black;
+  margin-right: 20px;
+  height: 60px;
+  align-items: center;
+}
+
+a:hover {
+  text-decoration: none;
+  color: #00acdc;
+}
+
+.a1 {
+  color: #00acdc;
+  position: relative;
+}
+
+.a4 {
+  margin: 0;
+}
+
+.a4:hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)